scripts/create_crosspog_json.py

```diff
@@ -5,9 +5,10 @@
 import json
 import os
 import yaml
+import math
 
 # set epsilon value to avoid giant scale factors
-epsilon = 0.0001
+epsilon = 0.001
```

Comment:
The problematic regions are resolved at the fit level. We could revert this value to 0.0001, however, from a (combined) efficiency of < 0.0001 it would yield a scale factor of < 10000. Reducing it by (at least one) order of magnitude seems to be a better option.

Reply:
ok, then we keep it that way, thanks !